src/database/models/likingAccommodation.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Likings = sequelize.define('Likings', {
```

Comment:
Great job @alexismajya  but liking is not a plural of like,
please change fix this

Reply:
cool

Reply:
Thank you Christian, for the feedback. however, according to our team naming conventions, the models should be singular

Reply:
yes that's why it should be named `like`